src/main/java/io/bastillion/manage/util/ExternalAuthUtil.java

```diff
@@ -225,7 +208,7 @@ public void handle(Callback[] callbacks) throws IOException,
                         }
 
                         //assign profiles for user
-                        UserProfileDB.assignProfilesToUser(con, user.getId(), allRoles, userInfo.getRoleNames());
+                        UserProfileDB.assignProfilesToUser(con, user.getId(), allRoles, allRoles);
```

Comment:
Is this correct? I would think it would still be 

`UserProfileDB.assignProfilesToUser(con, user.getId(), allRoles, userInfo.getRoleNames());``

Reply:
Please, see my reply regarding assignProfilesToUser below.